src/htcp.cc

```diff
@@ -1032,6 +1034,13 @@ HtcpReplyData::HtcpReplyData() :
     memset(&cto, 0, sizeof(cto));
 }
 
+int
+HtcpReplyData::parseHeader(const char *buffer, const size_t size)
+{
+    Http::ContentLengthInterpreter interpreter;
```

Comment:
Please add a comment (if you agree with it): `No applyStatusCodeRules() -- HTCP replies lack cached HTTP status code`.